-- Patch Notes Delivered Addon
-- Author: alvy023
-- File: patch-11.2.7.lua
-- Description: Patch notes text file for patch 11.2.7
-- License: License.txt
-- For more information, visit the project repository.

--- Export global notes variable
--- Max note length [140] ------------------------------------------------------------------------------------------------------------------
PatchNotesDelivered_Notes_1127 = {
    version = "11.2.5",
    build = "64502",
    hotfix = 1,
    gameChangesHotfixes = [[
        
    ]],
    gameChangesPatch = [[
        December 1, 2024

    With the Ghosts of K’aresh behind us, Azeroth stands on the precipice of a future overshadowed by Xal’atath and the relentless advance of the Void. In "The Warning", the stage is set for what comes next before continuing the story in the Midnight expansion. Players will also be welcomed home with the start of Housing Early Access!

CONTINUING THE WAR WITHIN CAMPAIGN

    • Though the Ghosts of K'aresh have been put to rest, a new danger looms. Vereesa Windrunner is stricken by visions of a new threat: a dire prophecy of Quel'Thalas' fall. You’ll continue The War Within campaign beginning with the Visions of a Shadowed Sun quest found in your Adventure Guide (Shift-J) to learn more about what portents await.

STAKE YOUR CLAIM WITH HOUSING EARLY ACCESS

    • Players who have purchased any edition of the Midnight expansion can join a neighborhood, purchase a house, and start customizing their new home with Housing Early Access! If you don’t have Midnight yet, don’t worry. You’ll still be able to begin adding to your Housing collection through a variety of activities and achievements, or by purchasing them from vendors throughout Azeroth.

NEW AND RETURNING PLAYER EXPERIENCES

    • Whether you’re a new player or returning after a hiatus, we’ve got you covered. New players will discover a smoother transition to get caught up with an updated New Player Experience. Players seeking to return to Azeroth will find a guided Returning Player Experience to help you remember how to play the game and get you on the path to the next latest content in World of Warcraft.

NEW LOREWALKING: THE ELVES OF QUEL'THALAS

    • Meet with Lorewalker Cho to learn about the origins of the Blood Elves and the Void Elves. You’ll begin your journey with a retelling of the Scourge attack on Silvermoon City and the Lich King’s assault on the Sunwell. Later, you’ll find out more about the tainting of the Sunwell by Alleria Windrunner, and the creation of the Void Elves.

PANDAREN HERITAGE ARMOR

    • Return to the Wandering Isle alongside a familiar cast of pandaren to set right all that’s gone awry with preparations for the Spirit Festival. Rally villagers and spirits alike to restore harmony to the celebration—and earn the new Pandaren Heritage Armor for your collection.

TAKE A RIDE ON TURBULENT TIMEWAYS DECEMBER 2 TO FEBRUARY 9

    • Once more, the Bronze Dragonflight has discovered growing disturbances in the timeways, with several rapidly intersecting our own! Players will be able to take on six new dungeons from the Shadowlands for the first time as a part of this limited-time ten-week event. Collect new rewards, earn new meta-achievements, and more.

THE BRAWLER'S GUILD RETURNS DECEMBER 9

    • Step back into the fighting ring to engage in one-on-one combat with variety of new and familiar bosses. Players will once more be able to sign up to take on these foes and climb the ranks or stand ring side and watch the spectacle.
        > Locations:
            + Orgrimmar: Brawl’gar Arena
            + Stormwind: Bizmo’s Brawlpub (Deeprun Tram)
    • We’re making some additional improvements, changing the boss rotation, and adding some new bosses along the way. Players will also be able to earn challenge achievements and collect some new rewards including Housing decor, a recolored flying mount, and even Bruce in a new color variation!

HORRIFIC VISIONS REVISITED

    • Players can now queue for either Orgrimmar or Stormwind Horrific Visions Revisited (was on a weekly rotation).

ITEMS

    • The following legacy currencies can now be losslessly shared between characters within a warband: Apexis Crystals (Warlords of Draenor), Order Resources (Legion), and Veiled Argunite (Legion).

SKYRIDING

    • Most abilities now have 6 charges and Vigor has been removed.
    • Take Off is now free to use.
    • Whirling Surge is now on a 30 second cooldown.

USER INTERFACE AND ACCESSIBILITY

    • Press and Hold option now works with the Single Button Assistant.
    • Guards at capital cities now offer the location of Training Dummies as a point of interest.
    • Default behaviors for A and D have been changed from turning to strafe. On login, existing players may be prompted with a choice to set their movement keys to use strafe or keep them to turn their character.

CATCH UP EXPERIENCE

    • If a player hasn't logged in for 60 days, on the character select screen, characters may have the option to log-in to the Catch Up Experience.
    • A new Tutorials tab has been added to the Adventure Guide for players to automatically start Catch Up Experience at any time.
    • Various gameplay tutorials have been added or updated.
    • Item comparison is now on by default and has had its UI improved to help with visual clarity.
    • Updated the art for the following Hand cursors:
        > Grabbing
        > Pointing
        > Holding
        > Open
    • The default keybind for the PvP Group Finder has been changed to SHIFT+N (was H).
    ]],
    addonChanges = [[
    ]],
}